package.json

```diff
@@ -2,10 +2,20 @@
   "name": "cross-domain-utils",
   "version": "1.0.9",
   "description": "Javascript module template.",
-  "main": "index.js",
+  "main": "dist/cross-domain-utils.js",
   "scripts": {
-    "test": "gulp test",
-    "postinstall": "node postinstall.js"
+    "postinstall": "flow-typed install",
+    "build": "babel-node ./node_modules/.bin/webpack --progress --output-path dist",
+    "karma": "babel-node ./node_modules/.bin/karma start",
+    "pretest": "npm run lint",
+    "test": "npm run karma",
+    "lint": "eslint src/ test/ && flow",
+    "clean": "rimraf dist coverage",
+    "reinstall": "rimraf node_modules && npm install",
+    "release:patch": "npm run build && npm version patch",
```

Comment:
You can.  I copied them from another project that I use and it works nice.  I don't do anything in npm for this project.